src/components/CardSchemes/CardSchemes.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+
+import styled, { css } from 'react-emotion';
+import PaymentMethodIcon from './components/PaymentMethodIcon';
+import { sizes } from '../../styles/constants';
+
+const { BYTE, KILO, MEGA, GIGA } = sizes;
+
+const CardSchemeBaseStyles = ({ theme }) => css`
+  box-sizing: content-box;
+  display: inline-block;
+  padding: ${theme.spacings.byte};
+`;
+
+const listWrapperBaseStyles = () => css`
+  display: block;
+  line-height: 0;
+  text-align: center;
+`;
+
+const CardSchemeWrapper = styled('ul')(listWrapperBaseStyles);
+const CardScheme = styled('li')(CardSchemeBaseStyles);
```

Comment:
Would we also want to make the `CardSchemes` component a `PaymentMethods` component? Not sure what the goal here is, exactly.